baselines/previously_seen_aws_cross_account_activity___dm.yml

```diff
@@ -0,0 +1,32 @@
+name: Previously Seen AWS Cross Account Activity - DM
+id: 0de7ce99-ab0a-41fe-9624-345df83f08cc
+version: 1
+date: '2020-05-28'
+description: This datamodel search looks for **AssumeRole** events where the requesting account
+  differs from the requested account, then writes these relationships to a lookup
+  file.
+how_to_implement: You must install the AWS App for Splunk (version 5.1.0 or later)
+  and Splunk Add-on for AWS (version 4.4.0 or later), then configure your CloudTrail
+  inputs. You must also be using CIM version x.x.x, which contains updates to the Authentication
+  data model for cloud use cases. Validate the user name entries in `previously_seen_aws_cross_account_activity.csv`,
+  a lookup file created by this support search.
+author: Rico Valdezy, Splunk
+search: '`cloudtrail` eventName=AssumeRole | spath output=requestingAccountId path=userIdentity.accountId
```

Comment:
We got two search keys in this manifest. Is that intentional?

Reply:
nope. Left the old one in by mistake. Next push will fix.